app/src/main/kotlin/com/simplemobiletools/smsmessenger/extensions/Context.kt

```diff
@@ -765,6 +766,16 @@ fun Context.getLockScreenVisibilityText(type: Int) = getString(
     }
 )
 
+fun Context.removeDiacriticsIfNeeded(text: String): String {
```

Comment:
there is a fun String.normalizeString() extension in Simple Commons doing this already, use that please. You will obviously still have to do the "config.useSimpleCharacters()" check in this app as Commons isnt aware of it